string robot_id
string robot_type
int8 x
int8 y
int8 theta
string current_action
bool is_blocked